proto/iotics/api/twin.proto

```diff
@@ -68,9 +68,21 @@ message ListAllTwinsRequest {
 message ListAllTwinsResponse {
```

Comment:
We should probably remove/amend the above comment. It's confusing in that a) we don't have the async version b) we have enforced limits (so couldn't "fail" like it says) and c) we'll soon have sparql with which you could "stream" all the twins if you really wanted to.